scripts/dotnet-cli-build/CompileTargets.cs

```diff
@@ -149,6 +149,11 @@ public static BuildTargetResult RestoreLockedCoreHost(BuildTargetContext c)
             return c.Success();
         }
 
+        [Target(
+            nameof(CompileCoreHost),
+            nameof(BuildCoreHostPackages))]
+        public static BuildTargetResult BuildCoreHost(BuildTargetContext c) => c.Success();
```

Comment:
We had to separate CompileCoreHost and BuildCoreHostPackages into two unrelated targets to enable signing of host and dependent libraries. This change will break that.